### ISSUE TITLE
InlineCompletion examples not working

### ISSUE THREAD

**Issue by juli1** [microsoft/vscode#156393]
The examples for the InlineCompletions are not working. See discussion here: https://github.com/microsoft/vscode-discussions/discussions/26

Steps to Reproduce:

1. Open the example
2. Try to make it run

Also, the inline completion API is not available in `@types/vscode`.

**Comment by juli1** [comment]
Hello,

Any update on this bug? I tried to check using the API with the last version of `@types/vscode` today, and it does not seem to include the API definition. I also tried to use the example of the [inline completion](https://github.com/microsoft/vscode-extension-samples/tree/main/inline-completions), and it does not work with the current version of VS Code.

Any update on when this feature will be available, documented with an example? I understand that may not be a priority, we are working on a new vscode plugin and the inline completion are a key feature for us, which is why we are asking if there is any idea of when this will be available.

Thank you!

**Comment by hediet** [comment]
> I tried to check using the API with the last version of @types/vscode today, and it does not seem to include the API definition.

Works for me:

![image](https://user-images.githubusercontent.com/2931520/182808798-6a6bb0cd-53a8-45e0-ad01-4e6228b6ae93.png)

"@types/vscode": "^1.69.0"

> Any update on when this feature will be available, documented with an example?

The feature is available right now.
We would welcome a PR to update the example!

**Comment by juli1** [comment]
Hi Henning,

Thanks for getting back to me! Let me try again then!

**Comment by juli1** [comment]
Hi Henning,

I checked and it works. I tried to specify a `range` in the `InlineCompletionItem` but every time I specify one, the suggestions are not picked up. Should I open a pull request?

I sent a pull request to add an example for inline completion that uses `@types/vscode` and works with the current version. See https://github.com/microsoft/vscode-extension-samples/pull/695

**Comment by hediet** [comment]
> Should I open a pull request?

You mean issue?

Please! But also provide a minimal sample to reproduce. Did you double check that the range and text you provide is correct and matches the documented constraints?

**Comment by hediet** [comment]
> I checked and it works. 

Can we close this issue then?

**Comment by juli1** [comment]
Yes, thank you for your patience!
